Debug support for dapr k8s services and daprd (#1381)
dapr/docs: daprdocs/content/en/developing-applications/debugging/_index.md

---
type: docs

> title: "Debug"

Reviewer: ```suggestion
title: "Debugging Dapr applications and the Dapr control plane"
```

Titles should be more descriptve than the linkTitles and be a full sentence

> linkTitle: "Debug"

Reviewer: ```suggestion
linkTitle: "Debugging"
```

> weight: 50

Reviewer: Weight should be 60 so it doesn't conflict with Middleware

```suggestion
weight: 60
```

> description: "How to debug Dapr application and Dapr itself"

Reviewer: ```suggestion
description: "Guides on how to debug Dapr applications and the Dapr control plane"
```
